code/src/main/java/com/google/sps/storage/data-storage/PositiveUserLocations.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.storage;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import java.time.Instant;
+import java.util.ArrayList;
+
+/**
+* Store information needed to keep track of a list of a positive user's locations from location history
+*/
+@Entity
+public class PositiveUserLocations {
+
+  @Id private String userId;
+  private ArrayList<Location> locations;
+  @Index private long timeCreatedSeconds;
+
+  // Objecify requires one constructor with no parameters
+  private PositiveUserLocations() {}
+
+  public PositiveUserLocations(String userId, ArrayList<Location> locations) {
+    this.userId = userId;
+    this.locations = locations;
+    timeCreatedSeconds = Instant.now().getEpochSecond();
+  }
+
+  public String getUserId() {
+      return userId;
```

Comment:
nit: inconsistent spacing.

Reply:
I hate that the default spacing in cloud shell if 4 spaces